src/extensions/default/BrambleUrlCodeHints/camera/interface.js

```diff
@@ -89,8 +90,23 @@ define(function (require, exports, module) {
                 return self.camera.fail();
             }
 
-            var binaryDataStr = /^data:image\/png;base64,(.+)/.exec(data)[1];
-            self.camera.savePhoto(base64ToBuffer(binaryDataStr));
+            var binaryDataStr;
+            if(filtered === true){  //added extra '=' for travis error
```

Comment:
As I say below, let's get rid of the filtered vs. non-filtered approach, and just have a single code path that always renders through Caman.

Another tip: you can do an `if` like this that checks for `true` without any operator: `if(filtered) {`